packages/create-fuels/src/cli.ts

```diff
@@ -64,6 +64,7 @@ export const runScaffoldCli = async (
   await mkdir(projectPath);
 
   await cp(join(__dirname, '../templates/nextjs'), projectPath, { recursive: true });
+  await rename(join(projectPath, 'gitignore'), join(projectPath, '.gitignore'));
```

Comment:
Did you consider keeping the previous filename and copy the file over?

```suggestion
  await cp(join(projectPath, '.gitignore'), join(projectPath, '.gitignore'));
```

Reply:
We cannot do that since this file will not be present in the bundle

Reply:
Only the `gitignore` (without the .) will be present